docs/developer/cluster-management-resources.md

```diff
@@ -0,0 +1,88 @@
+
+The nodes in Cluster Explorer are not to be confused with the machines that are shown in the Cluster Management app. In Cluster Management, if you go to a list of machines under **Advanced > Machines** in the side nav, you are not looking at nodes defined in the configuration of a single Kubernetes cluster. You are looking at a CAPI Machine resource that exists in the Rancher server's local cluster, which is also called the management cluster. These Machine resources declaratively specify the desired configuration of machines in the downstream clusters, including many hardware details that the downstream cluster is not aware of.
+
+To summarize, V1 provisioned clusters have nodes, nodePools and nodeTemplates. These are shown in the Cluster Management detail pages, which shows what is configured in the provisioning Cluster resource.
+
+In V2 cluster provisioning, clusters have Machines (https://cluster-api.sigs.k8s.io/user/concepts.html#machine) which are instances of MachineTemplates and have specific configuration for each infrastructure provider (Digital Ocean, Azure, etc). These infrastructure configuration details are shown in the Cluster Management detail pages for each cluster. In the Rancher UI, the resources used for configuring note templates, such as DigitaloceanConfigs, are hidden because every time they are edited, it forces all node pools using the config to reprovision, which could cause downtime to apps or services on the cluster.
+
+## Cluster Resources
+
+There are three Kubernetes resources in the local Rancher server Kubernetes cluster that are all named Cluster. Since the similar terminology can be somewhat confusing, here is a summary of how to differentiate them.
+
+### Management Clusters - `management.cattle.io.clusters`
+
+Rancher maintains a list of management clusters to maintain a consistent API for tracking all kinds of Kubernetes clusters, including imported clusters. There is one management Cluster resource for every downstream cluster managed by Rancher. It is also the Steve API's representation of Norman/v1 clusters.
+
+In the UI, the management Cluster resource is used in Cluster Explorer. like the provisioning and capi cluster resources. It’s used to show info on kubernetes version, cloud provider, node OSs, and resource usage. No, you can’t edit them, they’re available to users who can work within clusters but can’t necessarily provision clusters. rke1 and rke2 clusters both have management clusters; it’s like, a shared interface between all types of clusters so we can have a consistent experience within cluster explorer
+			* v1
+
+The kubectl command to get the management Cluster resources is:
```

Comment:
We should make sure that we're consistent with how `kubectl` is referenced. It's sometimes in a code block while it isn't in other places. 

```suggestion
The `kubectl` command to get the management Cluster resources is:
```